docs/rpc_nodes.md

```diff
@@ -18,13 +18,9 @@ author: Roxane Letourneau
 - ECAD Labs nodes:
     - Mainnet: https://mainnet.api.tez.ie
     - Hangzhounet: https://hangzhounet.api.tez.ie protocol `PtHangz2aRngywmSRGGvrcTyMbbdpWdpFKuS4uMWxg2RaH9i1qx`
-    - ~~Granadanet: https://granadanet.api.tez.ie protocol~~ ~~`PtGRANADsDU8R9daYKAgWnQYAJ64omN1o3KMGVCykShA97vQbvV`~~
-        - Granadanet has been replaced with Hangzhou.
-    - Idiazabalnet: https://idiazabalnet.ecadinfra.com/ protocol: `ProtoALphaALphaALphaALphaALphaALphaALphaALphaDdp3zK`
-        - Idiazabalnet is an early testnet protocol that will become the `I` proposal. It includes many features, the most notable being the new "tenderbake" consensus algorithim. 
+    - Ithacanet: https://ithacanet.ecadinfra.com/ protocol: `PsiThaCaT47Zboaw71QWScM8sXeMM7bbQFncK9FLqYc6EKdpjVP`
 - SmartPy public Tezos nodes
     - Mainnet: https://mainnet.smartpy.io
-    - Granadanet: https://granadanet.smartpy.io/
 - Blockscale public nodes operated on behalf of the Tezos Foundation: 
     - Mainnet: https://rpc.tzbeta.net/
     - Next protocol/version testnet (Granadanet): https://rpczero.tzbeta.net/
```

Comment:
Granadanet can be replaced with Ithacanet:
`Next protocol/version testnet (Ithacanet)`

Reply:
replaced